tools/serverpod_cli/bin/serverpod_cli.dart

```diff
@@ -177,6 +191,8 @@ Future<void> _main(List<String> args) async {
       bool verbose = results.command!['verbose'];
       String template = results.command!['template'];
       bool force = results.command!['force'];
+      int pgPort = int.parse(results.command!['postgresPort']);
```

Comment:
`postgresPort` for consistency. This should either result in a meaningful error message rather than throwing an exception on an invalid value. Or, a simpler solution could be to just use the default port, in which case you can use `int.tryParse`